std/special/compiler_rt/truncXfYf2.zig

```diff
@@ -0,0 +1,115 @@
+const std = @import("std");
+
+pub extern fn __gnu_f2h_ieee(a: f32) u16 {
+    return @bitCast(u16, truncXfYf2(f16, f32, a));
+}
+
+pub extern fn __truncsfhf2(a: f32) u16 {
+    return @bitCast(u16, truncXfYf2(f16, f32, a));
+}
+
+const CHAR_BIT = 8;
+
+pub fn truncXfYf2(comptime dst_t: type, comptime src_t: type, a: src_t) dst_t {
+    const src_rep_t = @IntType(false, @typeInfo(src_t).Float.bits);
+    const dst_rep_t = @IntType(false, @typeInfo(dst_t).Float.bits);
+    const srcSigBits = std.math.floatMantissaBits(src_t);
+    const dstSigBits = std.math.floatMantissaBits(dst_t);
+    const SrcShift = std.math.Log2Int(src_rep_t);
+    const DstShift = std.math.Log2Int(dst_rep_t);
+
+    // Various constants whose values follow from the type parameters.
+    // Any reasonable optimizer will fold and propagate all of these.
+    const srcBits: comptime_int = @sizeOf(src_t) * CHAR_BIT;
```

Comment:
You can omit all these explicit `comptime_int` declarations. I think it is clear enough that all the values are comptime implicitly.

Reply:
Done.